tools/src/icon4pytools/py2fgen/wrappers/diffusion.py

```diff
@@ -65,8 +67,64 @@ def profile_disable():
     stats.dump_stats(f"{__name__}.profile")
 
 
+def grid_init(
+    cell_starts: Field[[dims.CellIndexDim], int32],
```

Comment:
It's a good point. I moved it outside now, however this required building a bit more "infrastructure" around it, which makes sense since there are more things in this shared global state such as the profiler for example, or other granules.

Reply:
Also allows for cleaner separation of concerns.